### PR TITLE
TECH Stop working by side-effect of require

### PR DESCRIPTION
**Pull request by rossille**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -101,29 +96,19 @@ function getIntFromEnv(env, name, defaultValue) {
...
- */
-function startInstanceFromEnv(env) {
-  const delay = getIntFromEnv(env, 'BLOCKED_DELAY', 2000);
-  const threshold = getIntFromEnv(env, 'BLOCKED_THERSHOLD', 100);
-  const instance = new BlockedMonitor(delay, threshold);
-  return instance.start();
-}
-
-const singletton = startInstanceFromEnv(process.env);
+const singletton = new BlockedMonitor();
```
Comment (jean-k-cp) [inline index.js]: typo in singleton

File: index.js [hunk 2]
```diff
@@ -11,24 +11,21 @@ class BlockedMonitor {
...
   }
 
   /**
    * Starts the monitoring. The "warmup time" counts from here.
    *
-   * @returns {BlockedMonitor} the instance, for chainability.
+   * @returns {void}
    */
   start() {
+    this.delay = getIntFromEnv(process.env, 'BLOCKED_DELAY', 2000);
```
Comment (jean-k-cp) [inline index.js:26]: the constructor seemed a legit place to do this, but there is no actual impact so OK

**Review by jean-k-cp — APPROVED** [review]
typo otherwise LGTM

File: index.js (hunk 2 above)
Comment (rossille) [reply index.js:26]: actualy puting it here makes testing easier

File: index.js (hunk 1 above)
Comment (rossille) [reply index.js]: :ok:

**Pull request merged**